react/Rating/Rating.js

```diff
@@ -0,0 +1,52 @@
+import styles from './Rating.less';
+
+import React, { PropTypes } from 'react';
+
+import StarIcon from '../icons/StarIcon/StarIcon';
+import HalfStarIcon from './HalfStarIcon.svg';
+import Icon from '../icons/icon';
+import ScreenReaderOnly from '../Accessibility/ScreenReaderOnly';
+
+const getPercent = (rating, position) => Math.round(Math.min(Math.max(rating - position, 0), 1) * 100);
+
+const getStar = (percent, key, starClassName) => {
+  const props = {
+    key: `star-${key}`,
```

Comment:
The key doesn't have to be prefixed, it only has the be unique amongst its siblings.